core/store/store.go

```diff
@@ -107,9 +107,9 @@ func (wrapper *lazyRPCWrapper) Subscribe(ctx context.Context, channel interface{
 		return nil, err
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	newCtx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
I would instead change the parameter name `ctx` in the function signature to `_`, to indidcate that it's unused.